src/components/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import axios from 'axios'
+import { useState, useEffect } from "react";
+
+
+import Topics from "./Topics";
+import LatestArticles from "./LatestArticles/LatestArticles";
+
+
+const HomePage = () => {
+
+  const [allArticles, setAllArticles] = useState([])
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    axios
+      .get("http://localhost:9090/api/articles")
```

Comment:
Ideally this api call could and should be extracted into a separate folder of all your API functions. More than this though you need update the endpoint you are using to be the actual API NOT a local one! 😄